### DISCUSSION TITLE
Would the ability to disable downloading the episode title images help others?

### DISCUSSION THREAD

**Discussion by pphillips-sf** [trakt-tools/universal-trakt-scrobbler#182] [Ideas]
As I have a super slow DSL connection as my only internet it would be very helpful for me to be able to disable the plugin from trying to disable the title card images as the image downloads slows down the plugin GREATLY for me.

I was wondering if there were enough others that would find that useful to warrant it to be programmed into the plugin.

**Comment by rafaelgomesxyz** [comment]
There's an option to disable loading images in the latest beta version.
